products/devstudio/get-started.adoc

```diff
@@ -15,7 +15,7 @@ Prerequisites &#38; Required Dependencies Checklist
 
 == Step1 Content
 1.  Install the Java SE Development Kit (JDK) version 9. We recommend using the OpenJDK or the Oracle JDK.
-2.  Download link:#{site.download_manager_base_url}/download-manager/file/jboss-devstudio-10.0.0.GA-installer-eap.jar[JBoss Developer Studio 10.0.0.GA with JBoss EAP]
+2.  Download link:#{site.download_manager_base_url}/download-manager/file/devstudio-10.0.0.GA-installer-eap.jar[JBoss Developer Studio 10.0.0.GA with JBoss EAP]
 3.  Run the JBoss Developer Studio installer.
```

Comment:
This isn't in DM staging, so the link will show up as broken. It does work if you replace the url with production's.